samples/tricore-none-eabi/crosstool.config

```diff
@@ -0,0 +1,55 @@
+CT_CONFIG_VERSION="4"
+CT_EXPERIMENTAL=y
+CT_ARCH_TRICORE=y
+CT_ARCH_FLOAT_SW=y
+CT_TARGET_CFLAGS="-ftls-model=local-exec"
+CT_CC_LANG_CXX=y
+CT_TARGET_VENDOR="none"
+CT_BINUTILS_EXTRA_CONFIG_ARRAY="--disable-warn-rwx-segments"
+
+
+CT_GCC_V_13=y
+CT_GCC_VERSION="13.1.0"
+CT_GCC_SRC_DEVEL=y
+CT_GCC_DEVEL_VCS_git=y
+CT_GCC_DEVEL_VCS="git"
+CT_GCC_DEVEL_URL="https://github.com/EEESlab/tricore-gcc.git"
+CT_GCC_DEVEL_BRANCH="main"
+CT_GCC_DEVEL_REVISION="2a53638cc098057928f695947be94e5be3757598"
+CT_GCC_DEVEL_SUBDIR=""
+CT_GCC_DEVEL_BOOTSTRAP=""
+
+CT_BINUTILS_V_2_40=y
+CT_BINUTILS_VERSION="2.40.0"
+CT_BINUTILS_SRC_DEVEL=y
+CT_BINUTILS_DEVEL_VCS_git=y
+CT_BINUTILS_DEVEL_VCS="git"
+CT_BINUTILS_DEVEL_URL="https://github.com/EEESlab/tricore-binutils-gdb.git"
+CT_BINUTILS_DEVEL_BRANCH="master"
+CT_BINUTILS_DEVEL_REVISION=""
+CT_BINUTILS_DEVEL_SUBDIR=""
+CT_BINUTILS_DEVEL_BOOTSTRAP=""
+
+
+CT_NEWLIB_V_4_3=y
+CT_NEWLIB_VERSION="4.3.0"
+CT_NEWLIB_SRC_DEVEL=y
+CT_NEWLIB_DEVEL_VCS_git=y
+CT_NEWLIB_DEVEL_VCS="git"
+CT_NEWLIB_DEVEL_URL="https://github.com/EEESlab/tricore-newlib-cygwin.git"
+CT_NEWLIB_DEVEL_BRANCH="master"
+CT_NEWLIB_DEVEL_REVISION=""
```

Comment:
Could you set this to a specific commit id or tag. Otherwise when people build they will get a potentially different result.

Reply:
Done